MekHQ/data/universe/awards/standard.xml

```diff
@@ -0,0 +1,552 @@
+		<xp>3</xp>
+		<stackable>true</stackable>
+		<position>1.08.1</position>
+		<qty>4</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Scenario</range>
+	</award>
+	<award>
+		<name>Purple Heart</name>
+		<description>Wounded in combat due to enemy fire.</description>
+		<medal>PurpleHeartM.png</medal>
+		<ribbon>1-09-1-PurpleHeart.png</ribbon>
+		<stackable>true</stackable>
+		<position>1.09.1</position>
+		<qty>1</qty>
+		<item>Wounded</item>
+		<size>Individual</size>
+		<range>Event</range>
+	</award>
+	<award>
+		<name>House Meritorious Service</name>
+		<description>108 company kills in a mission.</description>
+		<medal>HouseMeritoriousServiceM.png</medal>
+		<ribbon>1-10-1-HouseMeritoriousService.png</ribbon>
+		<xp>3</xp>
+		<stackable>true</stackable>
+		<position>1.10.1</position>
+		<qty>108</qty>
+		<item>Kills</item>
+		<size>Company</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Meritourious Service</name>
+		<description>9 pilot kills in a mission.</description>
+		<medal>MeritoriousServiceM.png</medal>
+		<ribbon>1-10-2-MeritoriousService.png</ribbon>
+		<xp>2</xp>
+		<stackable>true</stackable>
+		<position>1.10.2</position>
+		<qty>9</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Combat Commendation</name>
+		<description>36 lance kills in a mission.</description>
+		<medal>CombatCommendationM.png</medal>
+		<ribbon>1-11-3-CombatCommendation.png</ribbon>
+		<xp>2</xp>
+		<stackable>true</stackable>
+		<position>1.11.3</position>
+		<qty>36</qty>
+		<item>Kills</item>
+		<size>Lance</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Combat Achievement</name>
+		<description>6 pilot kills in a mission.</description>
+		<ribbon>1-12-3-CombatAchievement.png</ribbon>
+		<xp>1</xp>
+		<stackable>true</stackable>
+		<position>1.12.3</position>
+		<qty>6</qty>
+		<item>Kills</item>
+		<size>Individual</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Combat Action</name>
+		<description>Involved in combat.</description>
+		<ribbon>1-13-1-CombatAction.png</ribbon>
+		<edge>1</edge>
+		<position>1.13.1</position>
+		<qty>1</qty>
+		<item>Combat</item>
+		<size>Individual</size>
+		<range>Career</range>
+	</award>
+	<award>
+		<name>House Unit Citation</name>
+		<description>216 battalion kills in a mission.</description>
+		<ribbon>2-01-2-HouseUnitCitation.png</ribbon>
+		<xp>3</xp>
+		<stackable>true</stackable>
+		<position>2.01.2</position>
+		<qty>216</qty>
+		<item>Kills</item>
+		<size>Battalion</size>
+		<range>Mission</range>
+	</award>
+	<award>
+		<name>Combat Unit Commendation</name>
+		<description>72 company kills in a mission.</description>
+		<ribbon>2-01-7-CombatUnitCommendation.png</ribbon>
+		<xp>2</xp>
+		<stackable>true</stackable>
+		<position>2.01.7</position>
+		<qty>72</qty>
+		<item>Kills</item>
+		<size>Company</size>
+		<range>Mission</range>
+	</award>
```

Comment:
Why was this completely changed from the original's 36 lance kills in a mission?